### PR TITLE
Add the entry point script for install all system packages

### PR DESCRIPTION
**Pull request by askeing**

### CODE REVIEW

**Comment by askeing** [comment]
ref: #35

File: scripts/install/all.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+MOZITP="/home/vagrant/MozITP"
```
Comment (shinglyu) [inline scripts/install/all.sh:3]: Can we have a global `config.sh` or something that contains this kind of global env variable. And in every script we can `source config.sh` to get them. 

You can see `MOZITP` appears in `scripts/install/system/provision.sh` again, and I believe we can use it in many other scripts as well.  The `config.sh` can contain things like default system path, default node/npm version, etc.

Comment (askeing) [inline scripts/install/all.sh:3]: agree

**Comment by askeing** [comment]
@shinglyu add `config.sh` seems like have some path issue, we should just merge this PR for entry point case, and then creat a new issue for config case.

**Comment by askeing** [comment]
@shinglyu r?

**Pull request merged**